src/lib/form-field/form-field.ts

```diff
@@ -195,6 +195,8 @@ export class MatFormField extends _MatFormFieldMixinBase
 
   _outlineGapStart = 0;
 
+  _initialGapCalculated = false;
```

Comment:
Rather than adding another property, couldn't we get away with checking that the gap is greater than 0?

Reply:
Currently that would cause a check loop for other types of form-field, since they set the gap to `0`. I could set it to `1` or some other number in that case since its not really used, but this just seemed clearer